### DISCUSSION TITLE
NVFLARE AND OS

### DISCUSSION THREAD

**Discussion by FabioNotaro2001** [NVIDIA/NVFlare#1262] [Q&A] [answered]
Hi everybody, I have a simple question: why NVFLARE framework needs an OS Unix-Like? Is there a reason? Maybe these OS are more suitable for machine learning and deep learning instead of Windows?

Thank you for your explanation.

**Comment by chesterxgchen** [comment, marked as answer]
Hi, thanks for you question. One should be can run it under Mac OS. Some training requires GPU, so linux makes it easier. Plus most of team developers only mac or linux machine.  Other reason is that some of dependency libraries does't work in other OS, for example, Homorphicencryption library used doesn't support mac OS.  

Which OS you like to use to run it ?

**Reply by FabioNotaro2001** [reply]
I'm currently running NVFLARE on a Linux distro for a reasearch work on federated learning and as you said the Linux environment makes many things easier, but I'm also a Windows user, so I just wondering why this framework has been developed and deployed only in Unix-like OSs. It was more a curiosity on what are tha advantages of this choice. In any case you answered my questions, so thank you very much!
